cypress/integration/26-register_publication/277-creator_navigates_to_summary_tab/summary.js

```diff
@@ -0,0 +1,104 @@
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+import { USER_WITH_AUTHOR } from '../../../support/constants';
+import 'cypress-localstorage-commands';
+import 'cypress-file-upload';
+
+const testFile = 'example.txt';
+
+// common steps
+
+Given('Creator begins registering a Registration in the Wizard', () => {
+  cy.loginCognito(USER_WITH_AUTHOR).then((idToken) => {
+    cy.wrap(idToken).as('idToken');
+    cy.setLocalStorage('i18nextLng', 'eng');
+    cy.setLocalStorage('previouslyLoggedIn', 'true');
+    cy.visit('/');
+    cy.get('[data-testid=new-publication]').click({ force: true });
+    cy.get('[data-testid=new-publication-file]').click({ force: true });
+    cy.get('input[type=file]').attachFile(testFile);
+    cy.get('[data-testid=publication-file-start-button]').click({ force: true });
+  });
+});
+
+When('they navigate to the Summary tab', () => {
+  cy.get('[data-testid=nav-tabpanel-submission').click({ force: true });
+});
+
+// common for Then

```

Comment:
skjønner ikke hva denne kommentaren betyr

Reply:
Tilsvarende som for kommentaren over så er det like steps som blir kjørt i begge scenarioene.